feat(content): Add external communications page (#79)
pyOpenSci/handbook: organization/external-communications.md

# External Communications

pyOpenSci uses a multitude of platforms for external communication, in order to both maintain communication with our current community, as well as grow our community. Our external communications fall under the broad categories of email, the pyOpenSci blog, pyOpenSci newsletters, and social media platforms. This section details each of those broad categories, as well as the specific platforms used within each, and provides relevant links and insights into the purpose, strategy, and associated processes for each platform.

## Email

pyOpenSci does not send unsolicited bulk emails, and all emails contain an unsubscribe link. Emails from pyOpenSci are sent to communicate with potential volunteers, share the monthly version of the newsletter, and to keep registered event participants up-to-date on logistics. pyOpenSci will also reach out to event attendees with pre- and/or post-surveys, along with follow-up reminders, as needed.

> ### Mailchimp

Author: it didn't - this PR went up before then! I'll go through and make adjustments.

> ### Mailchimp

Reviewer: Fantastic. thanks!

### EventBrite

pyOpenSci uses [EventBrite](https://eventbrite.com) to manage event sign-ups. Anyone who has registered for a pyOpenSci event through EventBrite will be contacted through the EventBrite service regarding event updates and logistics. pyOpenSci may also upload EventBrite contacts to Mailchimp or SurveyMonkey for the purposes of sending out pre- and/or post-surveys.

### SurveyMonkey

[SurveyMonkey](https://www.surveymonkey.com/) is used to send out pre- and/or post-surveys to attendees of pyOpenSci events. This can include in-person or online events such as workshops, talks, and sprints. SurveyMonkey is used due to its ease of use, integration with HubSpot, and ability to send out survey reminder emails.

### HubSpot

> [HubSpot](https://hubspot.com/) is pyOpenSci's customer relationship management (CRM) platform. Although HubSpot has email capabilities, pyOpenSci does not currently use this functionality.

Author: yep! related to the above, the PR was written and submit before we discussed these changes.

* Sharing news and updates related to pyOpenSci
* Build domain authority through SEO strategies

### How to add a new blog post

There are two main groups of people who submit blog posts to the pyOpenSci blog: pyOpenSci employees, and package authors/maintainers for packages that have been accepted into the [pyOpenSci ecosystem](https://www.pyopensci.org/python-packages.html). If you are not in one of these two groups, but would like to submit a blog post to pyOpenSci, please reach out to us at [media@pyopensci.org](mailto:media@pyopensci.org) to discuss your post prior to writing it and submitting a PR.

**Writing a blog post**

The following processes for writing a blog post should be used by both pyOpenSci employees as well as guest posts:
1. Fork the pyOpenSci website repo: [https://github.com/pyOpenSci/pyopensci.github.io](https://github.com/pyOpenSci/pyopensci.github.io).
2. Work on your blog post locally. Be sure to include alt text as needed, and create an author page if you do not already have one.
3. Submit a pull request from your fork of the website repo to [https://github.com/pyOpenSci/pyopensci.github.io](https://github.com/pyOpenSci/pyopensci.github.io).

> 4. Make sure that the blog post passes CI checks that look for spelling errors, broken links and more. Please note that occasionally the `htmlproofer` check will fail because the URL does not exist yet. We have implemented a fix for this, but it occasionally does not work.

Author: will do! I'm going to submit this PR for review, and then once the htmlproofer PR is complete and merged, will submit a patch to link to it here.

5. Request review of the blog post. A member of the pyOpenSci staff should always review any new content being added to the website.

> 6. Once you have an approving review, the pull request can be merged. In the case of guest blog posts, a member of the pyOpenSci team will merge your post.

Reviewer: ```suggestion
6. Once you have an approving review, the pull request can be merged. In the case of guest blog posts, a member of pyOpenSci's staff or contributor team will merge your post.
```

> pyOpenSci has GitHub organization-wide policies in place that ensure all pull requests are generally reviewed prior to being merged. In some instances we may bypass those if the PR is a small fix or critical update. In most cases a review is preferred. Please contact the Executive Director and Founder, Leah Wasser, with any questions or needs for clarification.

Reviewer: ```suggestion
pyOpenSci has GitHub organization-wide policies in place that require all pull requests are reviewed before being merged. In some instances, we may bypass those if the PR is a small fix or critical update. In most cases a review is preferred. 
```

Reviewer: Please do not add language to the handbook that says  "ask Leah". If you are unsure about the policy i'm super happy to chat about it.  But for now, I'm comfortable with saying something like "ask a member of the pyOS staff".

> **A note on guest blog posts**
> 
> Anyone who has had their package accepted into the pyOpenSci ecosystem is encouraged to submit a blog post (or series of blog posts) about their review experience and/or their package. When submitting your blog post, please include the following YAML elements:
> 
> ```
> layout: single
> title: "The title of your blog post"
> excerpt: "One to five sentences summarizing your post. This text will appear on our [blog landing page](https://pyopensci.org/blog)"
> author: "Your name"
> permalink: /blog/url-for-your-post.html
> header:
>   overlay_image: images/headers/your-blog-post-header-image.png
>   overlay_filter: rgba(20, 13, 36, 0.8)
> categories:
>   - blog-post
>   - community
> classes: wide
> toc: true
> comments: true
> ---
> ```

Reviewer: ```suggestion

:::{admonition} Yaml elements required for blog posts
:class: tip 

All package authors, maintainers (and users) of a pyOpenSci-accepted package are welcome and encourage to submit a blog post (or series of blog posts) about their review experience and/or the use of the package. When submitting your blog post, please include the following YAML elements:

```
layout: single
title: "The title of your blog post"
excerpt: "One to five sentences summarizing your post. This text will appear on our [blog landing page](https://pyopensci.org/blog)"
author: "Your name"
permalink: /blog/url-for-your-post.html
header:
  overlay_image: images/headers/your-blog-post-header-image.png
  overlay_filter: rgba(20, 13, 36, 0.8)
categories:
  - blog-post
  - community
classes: wide
toc: true
comments: true
---
```
:::
```

Reviewer: i just made this correction here - please check the build to ensure it's correct!

For more information on how pyOpenSci uses GitHub, please refer to the [GitHub section of our handbook](https://www.pyopensci.org/handbook/community/github/intro.html).

### Promoting blog posts

All blog posts should be promoted, regardless of whether or not they were written by a guest or a pyOpenSci employee! For promotion, coordinate with the Community Manager to ensure that there’s a tailored message for each of the following platforms:
* Slack
* Discourse
* BlueSky
* LinkedIn
* Fosstodon

> In addition to pyOpenSci promotions, we monitor social media for any personal posts authors have created to promote their post, boost accordingly.
> 

Reviewer: ```suggestion
In addition to pyOpenSci social outreach, we monitor social media for any personal posts that authors have created. To promote these posts, we will repost.

```

Reviewer: i'm not sure the intent of this sentence but I tried to rewrite. Please rewrite as you see fit.

## Newsletters

We publish two newsletters:
* A weekly newsletter on LinkedIn, published on Thursdays

> * A monthly newsletter published on LinkedIn and sent to all Mailchimp subscribers

Reviewer: Flaggin mailchimp. If we made a decision after august 14 that i've forgotten you can remind me!

> The weekly LinkedIn newsletter has three main categories:

Reviewer: This is great! Please add an example of how each of these posts is structured . 

then please also add what metrics are used to track success of these items. each section should include goals of the platform, target audience, metrics of success.
